.github/workflows/ci.yml

```diff
@@ -63,7 +62,13 @@ jobs:
 
   build-info:
     name: "Build info"
-    runs-on: ubuntu-20.04
+    runs-on: >-
```

Comment:
I think this can be replaced by a pre-requisite step (before build-info) that could produce a string output that will be used as "fromJson()" in the "runs-on". I can make a fixup with it.

Reply:
I thought about that but there is one major downside: that would have to run on "public" queues, meaning it would still be at the mercy of long queue time

Reply:
Not really. See https://github.com/astronomer/airflow/pull/1197

Reply:
Ohhh I see, yeah cool.

Reply:
Done in https://github.com/apache/airflow/pull/13730/commits/939a7fe948f2d917d1c1eca5100a1161cd5ab1b3

I kept it in the "Build Info" job, as each job needs a new worker, so having a worker tied up to run a very simple shell script seemed wasteful, especially as this falls reasonably cleanly under the "work out build info" description of the existing step

Reply:
Yep. Easier indeed.